manifests/stage-macvlan-cni/0010-macvlan-ds.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: macvlan-cni-ds
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    tier: node
+    app: macvlan-cni
+spec:
+  selector:
+    matchLabels:
+      name: macvlan-cni
+  template:
+    metadata:
+      labels:
+        name: macvlan-cni
+        tier: node
+        app: macvlan-cni
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
```

Comment:
in a real deployment, we dont want this to be deployed on the master node IMO

Reply:
changed to not deploy on master